app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
add .idea, build and .gradle dirs to this file along with other files like local.properties...

Reply:
Ok thanks!